packages/data/src/export/actions.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * External dependencies
+ */
+import { apiFetch } from '@wordpress/data-controls';
+
+/**
+ * Internal dependencies
+ */
+import TYPES from './action-types';
+import { NAMESPACE } from '../constants';
+
+export function setExportId( exportType, exportArgs, exportId ) {
+	return {
+		type: TYPES.SET_EXPORT_ID,
+		exportType,
+		exportArgs,
+		exportId,
+	};
+}
+
+export function setIsRequesting( selector, selectorArgs, isRequesting ) {
+	return {
+		type: TYPES.SET_IS_REQUESTING,
+		selector,
+		selectorArgs,
+		isRequesting,
+	};
+}
+
+export function setError( selector, selectorArgs, error ) {
+	return {
+		type: TYPES.SET_ERROR,
+		selector,
+		selectorArgs,
+		error,
+	};
+}
+
+export function* startExport( type, args ) {
+	yield setIsRequesting( 'startExport', { type, args }, true );
+
+	try {
+		const result = yield apiFetch( {
+			path: `${ NAMESPACE }/reports/${ type }/export`,
+			method: 'POST',
+			data: {
+				report_args: args,
+				email: true,
+			},
+		} );
+
+		if ( result && result.status === 'success' ) {
```

Comment:
I was pretty sure that `apiFetch` returns a [Response](https://developer.mozilla.org/en-US/docs/Web/API/Response)? As far as I can tell, the data control calls the `@wordpress/api-fetch` package
directly.

`Response.status` does not contain a string, but a HTTP status code like 200. Could you confirm or deny if thats the case here?

Also I notice that apiFetch will [throw on anything thats not 200 or 300](https://github.com/WordPress/gutenberg/blob/master/packages/api-fetch/src/index.js#L57) so I wonder if perhaps a `catch` is sufficient here?